docs/unified/hardkernel/recovery.md

```diff
@@ -0,0 +1,69 @@
+# Recovery
```

Comment:
This platform can also be recovered by dual bios jumper: https://wiki.odroid.com/odroid-h4/hardware/restore_h4_bios
It would be great to have documentation of the use of that mechanism in simplified form here.

Reply:
Would that suffice? Should I include an URL to the recovery instructions on the ODROID wiki?
https://github.com/Dasharo/docs/pull/903/commits/5c5403d99696a29d01195b2b1aa2987bbbda12c0